src/Compilers/CSharp/Portable/Compilation/SyntaxTreeSemanticModel.cs

```diff
@@ -2427,6 +2420,7 @@ internal override Symbol RemapSymbolIfNecessaryCore(Symbol symbol)
                         case SymbolKind.NamedType:
                             Debug.Assert((object)declaredSymbol.GetSymbol() == (object)ctor.ContainingSymbol);
                             // Accept nodes that do not match a 'parameter list'/'base arguments list'.
+                            //return (node) => true; 
```

Comment:
>//return (node) => true;  [](start = 28, length = 25)

We do not commit commented out code without a PROTOTYPE comment or an issue tracking the follow-up